pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -119,7 +117,7 @@ func (m *MultiNodeConsolidation) firstNConsolidationOption(ctx context.Context,
 	lastSavedCommand := Command{}
 	lastSavedResults := scheduling.Results{}
 	// Set a timeout
-	timeout := m.clock.Now().Add(MultiNodeConsolidationTimeoutDuration)
+	timeout := m.clock.Now().Add(options.FromContext(ctx).SinglenodeConsolidationTimeout)
```

Comment:
Fixing this might fix the failing test - not sure 🤔 https://github.com/kubernetes-sigs/karpenter/blob/main/pkg/controllers/state/suite_test.go#L1469, specifically [this](https://github.com/kubernetes-sigs/karpenter/blob/main/pkg/controllers/state/suite_test.go#L1469)